articles/openstack/ostack-how-install-rhui.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How to install Red Hat Update Infrastructure on an existing OpenStack instance | UKCloud Ltd
+description: Shows you how to update your existing OpenStack Instances to target UKCloud's approved Red Hat Update Infrastructure (RHUI)
+services: openstack
+author: Sue Highmoor
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Install Red Hat Update Infrastructure on an existing OpenStack instance
+toc_fullpath: How To/ostack-how-install-rhui-vm.md
```

Comment:
Actual file name seems to be ostack-how-install-rhui.md (i.e., no -vm at the end). Could you change here and in the next line please?

Reply:
Done